packages/fxa-content-server/tsconfig.json

```diff
@@ -43,9 +43,9 @@
 
         /* Module Resolution Options */
         "moduleResolution": "node" /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */,
-        "baseUrl": "./app/scripts" /* Base directory to resolve non-absolute module names. */,
+        // "baseUrl": "./app/scripts" /* Base directory to resolve non-absolute module names. */,
```

Comment:
Delete?

Reply:
left in since all other options are commented out in file